### ISSUE TITLE
Can not open "Statistic/Sync Load QPS" in Granafa

### ISSUE THREAD

**Issue by you06** [pingcap/tidb#34514]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

Start a minimal cluster with monitor, open the TiDB dashboard.

```sh
tiup playground nightly --tiflash=0
```

View the "Statistic/Sync Load QPS" panel.

![image](https://user-images.githubusercontent.com/9587680/167533451-283c40a1-2df6-424e-a848-13e523de060e.png)

### 2. What did you expect to see? (Required)

The "Statistic/Sync Load QPS" panel is displayed.

### 3. What did you see instead (Required)

The assertion panel is displayed and the size is not correct.

![image](https://user-images.githubusercontent.com/9587680/167533537-ce2be680-56df-4584-b036-f089c25e5ae9.png)

### 4. What is your TiDB version? (Required)

```
Release Version: v6.1.0-alpha
Edition: Community
Git Commit Hash: c16291392a4f06a82dfdc7184ab4917bb4b335ac
Git Branch: heads/refs/tags/v6.1.0-alpha
UTC Build Time: 2022-05-09 14:58:20
GoVersion: go1.18
Race Enabled: false
TiKV Min Version: v3.0.0-60965b006877ca7234adaced7890d7b029ed1306
Check Table Before Drop: false
```

**Comment by you06** [comment]
Panels with duplicated ids lead to this problem, and developers may edit the grafana file([`tidb.json`](https://github.com/pingcap/tidb/blob/ede6f8cf16a16da62609cd88296666eb8c42d10a/metrics/grafana/tidb.json)) individually, then the same id is selected and it works in both develop branch. The PRs also merge without conflict. Maybe we need some CI tests to check the grafana file is OK.

**Comment by ekexium** [comment]
> Panels with duplicated ids lead to this problem. Developers may edit the grafana file([`tidb.json`](https://github.com/pingcap/tidb/blob/ede6f8cf16a16da62609cd88296666eb8c42d10a/metrics/grafana/tidb.json)) individually, then the same id is selected and it works in both develop branch. The PRs also merge without conflict. Maybe we need some CI tests to check the grafana file is OK.

Methods in https://github.com/pingcap/tidb/pull/32333 might be a solution

**Comment by you06** [comment]
Fixed by #34513.
